lib/cdo/test_flakiness.rb

```diff
@@ -118,4 +132,10 @@ def self.recommend_reruns(flakiness)
       (scenario + feature) / 2.0
     end
   end
+
+  cached def self.from_timestamp
+    return nil unless File.exist?(FLAKINESS_TIMESTAMP_FILENAME)
+    timestamp = JSON.parse(File.read(FLAKINESS_TIMESTAMP_FILENAME))['timestamp']
+    timestamp.is_a?(Integer) ? timestamp : nil
+  end
```

Comment:
I would propose that we fail here if the timestamp is in the future. Otherwise, if a developer machine with a bad timestamp somehow set this for a year from now, our system would make it look like we didn't have any flaky tests.

Reply:
good call. i added [this commit](https://github.com/code-dot-org/code-dot-org/pull/36051/commits/2116035723c43274304528787847382ee9f9a0ad) to address that